src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -1136,7 +1136,7 @@ def load_arguments(self, _):
             c.argument('subnet_address_prefix', help='The subnet IP address prefix to use when creating a new VNet in CIDR format.')
             c.argument('nics', nargs='+', help='Names or IDs of existing NICs to attach to the VM. The first NIC will be designated as primary. If omitted, a new NIC will be created. If an existing NIC is specified, do not specify subnet, VNet, public IP or NSG.')
             c.argument('private_ip_address', help='Static private IP address (e.g. 10.0.0.5).')
-            c.argument('public_ip_address', help='Name of the public IP address when creating one (default) or referencing an existing one. Can also reference an existing public IP by ID or specify "" for None (\'""\' in Azure CLI using PowerShell or --% operator). For Azure CLI using powershell core edition 7.3.4, specify '' or "" (--public-ip-address '' or --public-ip-address "")')
+            c.argument('public_ip_address', help='Name of the public IP address when creating one (default) or referencing an existing one. Can also reference an existing public IP by ID or specify "" for None (\'""\' in Azure CLI using PowerShell or --% operator). For Azure CLI using PowerShell core edition 7.3.4, specify \'""\' (--public-ip-address \'""\')')
```

Comment:
Not sure why `For Azure CLI using PowerShell core edition 7.3.4, xxxx` is introduced. It's redundant and can be omitted as it repeats the previous sentence.

Reply:
The reason why I didn't remove the `For Azure CLI using PowerShell core edition 7.3.4, xxxx` sentence is that I'm not sure about the behavior of versions prior to `7.3.4`. The `--% operator` doesn't seem to work for `7.3.4`, but not sure if it works for the previous versions.